### PR TITLE
Remove unwanted try block inside createAppVersionInDataBase() and  cr…

### PR DESCRIPTION
**Pull request by kasundsilva**
…eateAppInDataBase() catch blocks

### CODE REVIEW

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 1]
```diff
@@ -266,49 +265,30 @@ var createAppInDataBase = function(applicationName, versionName, applicationDesc
...
-        } catch (e1){
-            log.error("Error while updating Application Creation Failed event for application : " + applicationName
-                              + " version : " + versionName);
-            log.error(e1);
-            // not throwing the error, because it will hide the actual exception
-        }
-
         log.error(msg);
         log.error(e);
         log.info("Application creation failed because initial application metadata persistence failed and roll back process" +
```
Comment (mcvidanagama) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: make this to log.warn

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag [hunk 2]
```diff
@@ -266,49 +265,30 @@ var createAppInDataBase = function(applicationName, versionName, applicationDesc
...
                  " is started for application : "+applicationName+" with version : "+ versionName);
-        deleteApplicationUploadedPath(versionHashId);
+        //Not deleting the uploaded artifact, we need to handle the retry logic
```
Comment (mcvidanagama) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: Make this more clear

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 1 above)
Comment (kasundsilva) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: https://github.com/wso2/app-cloud/pull/703

File: modules/jaggeryapps/appmgt/src/modules/application/application.jag (hunk 2 above)
Comment (kasundsilva) [inline modules/jaggeryapps/appmgt/src/modules/application/application.jag]: https://github.com/wso2/app-cloud/pull/703

**Pull request merged**